src/FSharp.MetadataFormat/FSharp.MetadataFormat.fsproj

```diff
@@ -119,7 +119,7 @@
     </ProjectReference>
   </ItemGroup>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.0')">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.0'">
```

Comment:
There is no guarantee that we can even combine two conditions... and it sounds like something quite hard to do.

Note that the reason for this particular change is a bugfix in paket which would generate different conditionals only for net40 (added additional parenthesis). I don't think we should pay any price to keep the diff small (ie. rather make the bugfix than to keep the quirk).